app/src/test/shared/components/OneTimeLinkContextTests.tsx

```diff
@@ -38,25 +38,31 @@ describe("OneTimeLinkContext", () => {
     }
     const Class = OneTimeLinkContext(EmptyComponent);
 
-    it("if store does not contain matching token, href passed to child is null", () => {
+    it("if store does not contain matching token, href passed to child is null", (done: DoneCallback) => {
         const rendered = shallow(<Class href="/orange/"/>, {context: {store}}).dive();
         const child = rendered.find(EmptyComponent);
-        expect(child.prop("href")).to.equal(null);
+        checkAsync(done, () => {
+            expect(child.prop("href")).to.equal(null);
+        });
     });
 
-    it("can get properties from store with matching token", () => {
+    it("can get properties from store with matching token", (done: DoneCallback) => {
```

Comment:
we were getting weird errors here due to asynchronous method being fired and not resolved before the test finished